CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
-## 2.22.1 (Unreleased)
+## 2.22.2 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.7.1324**
+- Supported Terraform version: **v1.x**
+
+### Features:
+#### Multi-Cloud Transit:
+1. Implemented support for enabling/disabling Jumbo Frames on GRE tunnels under BGP connections in **aviatrix_transit_external_device_conn**:
+   - ``enable_jumbo_frame``
+
+### Enhancements:
+1. Added duplicate rules check and remove deprecation message for ``domain_names`` in **aviatrix_fqdn** to continue support it and the standalone **aviatrix_fqdn_tag_rule** resource as well
+2. Added duplicate rules check and Remove deprecation message for ``policy`` in **aviatrix_firewall** to continue support it and the standalone **aviatrix_firewall_policy** resource as well
```

Comment:
Please follow exact same template for the firewall one as well, as described above for FQDN